opentech/apply/projects/templates/application_projects/project_admin_detail.html

```diff
@@ -6,7 +6,44 @@
 {% block admin_sidebar %}
 <div class="modal" id="send-for-approval">
     <h4 class="modal__header-bar">Request Approval</h4>
+
+    {% if not remaining_document_categories %}
     {% include 'funds/includes/delegated_form_base.html' with form=request_approval_form value='Request'%}
+    {% else %}
```

Comment:
What is to stop us just injecting the text before the form?

It might make sense to add a `cancel` boolean to the `delegated_form_base` to include a cancel button.

```django
{% if not remaining_document_categories %}
        <h5>Are you sure you're ready to upload?</h5>

         <p>This project is missing the following documents:</p>

         <ul>
            {% for missing in remaining_document_categories %}
            <li><strong>{{ missing.category.name }} ({{ missing.difference }})</strong></li>
            {% endfor %}
        </ul>
{% endif %}  
{% include 'funds/includes/delegated_form_base.html' with form=request_approval_form value='Request'%}
```